XenAdmin/Diagnostics/Problems/PoolProblem/PoolHasFCoESrWarning.cs

```diff
@@ -28,58 +28,33 @@
  * SUCH DAMAGE.
  */
 
-using System;
-using System.Diagnostics;
-using System.Reflection;
-using XenAdmin.Actions;
 using XenAdmin.Core;
 using XenAdmin.Diagnostics.Checks;
-using XenAdmin.Dialogs;
 using XenAPI;
 
 
 namespace XenAdmin.Diagnostics.Problems.PoolProblem
 {
     internal class PoolHasFCoESrWarning : WarningWithMoreInfo
     {
-        private static readonly log4net.ILog Log = log4net.LogManager.GetLogger(MethodBase.GetCurrentMethod()?.DeclaringType);
         private readonly Pool _pool;
         private readonly bool _upgradingToVersionWithDeprecation;
 
-        public override string Title => Check.Description;
-        public override string HelpMessage => Messages.MORE_INFO;
-        public override string LinkText => Messages.LEARN_MORE;
-        public override string LinkData => InvisibleMessages.PV_GUESTS_CHECK_URL;
-        public override string Message => string.Empty;
+        public override string Title => Description;
 
-        public override string Description => string.Format(_upgradingToVersionWithDeprecation ? Messages.POOL_HAS_DEPRECATED_FCOE_WARNING : Messages.POOL_MAY_HAVE_DEPRECATED_FCOE_WARNING,
-            _pool,
-            BrandManager.ProductVersionPost82
-        );
-        
-        public PoolHasFCoESrWarning(Check check, Pool pool, bool upgradingToVersionWithDeprecation) : base(check)
+        public override string Message => string.Format(_upgradingToVersionWithDeprecation

```

Comment:
I like this solution but we're not linking to the docs (except for when viewing the deprecation notice from other parts of the app). Is this ok? Can we have both?

Reply:
They should be accessible from the link on the pop-up.

Reply:
The link is missing 🤦